app/lib/database/utils.test.ts

```
@@ -1,10 +1,9 @@
-import type { Q } from '@nozbe/watermelondb';
+import type { Q } from './facade';
```

ESLint

[error] There should be no empty line within import group

(import/order)